python/ray/data/_internal/logging.py

```diff
@@ -1,14 +1,13 @@
 import logging
+import logging.config
 import os
 from typing import Optional
 
-import ray
-from ray._private.log import PlainRayHandler
-from ray._private.ray_constants import LOGGER_FORMAT
+import yaml
 
-_default_file_handler: logging.Handler = None
+import ray
 
-DEFAULT_DATASET_LOG_FILENAME = "ray-data.log"
+CONFIG_FILENAME = "logging.yaml"
```

Comment:
let's make this file path configurable ( can be as simple as an env var), and add a test with a custom logging.yaml file.